### PR TITLE
[MI-2813]: Added a placeholder for default selected option in autoselector

### PR DESCRIPTION
**Pull request by Nityanand13**
Screenshot from 2023-02-24 13-44-30

### CODE REVIEW

File: src/styles/components/_autoSuggest.scss [hunk 1]
```diff
@@ -38,12 +38,12 @@
     }
 
     &--shifted {
-      padding-top: 10px;
+      padding-top: 13px;
```
Comment (sooraj-shukla) [inline src/styles/components/_autoSuggest.scss:41]: Why is this required? Wasn't it working fine?

Comment (Nityanand13) [reply src/styles/components/_autoSuggest.scss:41]: It was congested earlier so updated that.

**Review by sooraj-shukla — DISMISSED** [review]

**Review by manojmalik20 — APPROVED** [review]
